File: src/test/samples/invalidSamples/invalidSample.go
package main

import "fmt"

//#region FirstRegion
var x = 42
//#endregion

// #endregion Invalid end boundary
// #region Invalid start boundary

// #region      Second Region
type MyClass struct{}

// #region InnerRegion
func (m MyClass) myMethod() {}
        //        #endregion ends InnerRegion

        // #region
func (m MyClass) myMethod2() {}
//#endregion

// #endregion

func main() {
    fmt.Println("Sample")
}

File: src/test/samples/invalidSamples/invalidSample.cs
// #region FirstRegion
int x = 42;
// #endregion

// #endregion Invalid end boundary
// #region Invalid start boundary

// #region Second Region
class MyClass {
    //  #region InnerRegion  
    void MyMethod() {}
    //#endregion   ends  InnerRegion 

    // #region
    void MyMethod2() {}
    //    #endregion  
}
// #endregion
